contrib/epee/include/net/http_protocol_handler.inl

```diff
@@ -543,16 +543,21 @@ namespace net_utils
 	bool simple_http_connection_handler<t_connection_context>::handle_request_and_send_response(const http::http_request_info& query_info)
 	{
 		http_response_info response;
```

Comment:
This change just made me think of some more annoying things about this epee server...

`http_response_info` unfortunately does not have a constructor. There should be some uninitialized value issues now, since the type has some integer members. I think its worth slipping in `http_response_info response{}` on this line which value initializes all members (the integers to 0). I would then add an else block to set `response.m_response_code = 200` and `response.m_response_comment = "OK"`.

Then, there is still one [strange oddity with `Content-Type`](https://github.com/monero-project/monero/blob/master/contrib/epee/include/net/http_protocol_handler.inl#L594). I would add a `if (!response.m_mime_tipe.empty()) { ... }` so that the field is only added if there is an actual value to write. This would technically be a cleanup of an existing issue, but it seems like an appropriate time to clean it up.

Reply:
All set, sorry about that. I ran the options header again before but completely missed it not returning the status code. 

Should be set now -- I also made the misc change to wrap the content-type with a check for if it was empty so should be set there as well.